app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>DRIVERS</h1>
+<table>
+<tr><td>Driver Name</td><td>Rating</td></tr>
+<% Driver.active_drivers.each do |driver| %>
+<tr>
+  <td><a href="/drivers/<%= driver.id %>" name="<%= driver.name%>"><%= driver.name%></a></td>
```

Comment:
use `link_to` over anchor tags. Also use named paths rather than hard-coding the paths.